.gitlab-ci.yml

```diff
@@ -118,3 +122,42 @@ deploy-tomcat-production-fallback:
         - package-production
     only:
         - tags
+
+# Not executed, parent job definition for deployments of jar files
+.deploy-jar:
+    stage: deploy
+    image: alpine:3.9.5
+    environment:
+        name: $ENVIRONMENT_NAME
+    script:
+        # Start ssh-agent and set identity then copy the Jar file and create a symlink
+        # See https://docs.gitlab.com/ee/ci/ssh_keys/
+        - apk add --update openssh-client
+        - eval "$(ssh-agent)"
+        - echo "$EVA_ETL_PRIVATE_KEY" | tr -d '\r' | ssh-add -
+        - TMP_NAME=$(basename $(ls $CLUSTERING_ARTIFACT_PATH))
+        - CLUSTERING_ARTIFACT_NAME=${TMP_NAME%%.jar}-${ENVIRONMENT_NAME}.jar
+        - scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null $CLUSTERING_ARTIFACT_PATH ${CLUSTER_USER}@${CLUSTER_LOGIN_NODE}:${CLUSTERING_PATH}/${CLUSTERING_ARTIFACT_NAME}
+        - ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ${CLUSTER_USER}@${CLUSTER_LOGIN_NODE} "ln -fns ${CLUSTERING_PATH}/${CLUSTERING_ARTIFACT_NAME} ${CLUSTERING_PATH}/eva-accession-clustering.jar"
+
+deploy-jar-development:
+    extends: .deploy-jar
+    variables:
+        ENVIRONMENT_NAME: development
+        CLUSTERING_PATH: $CLUSTERING_DEVELOPMENT_PATH
+    dependencies:
+        - package-development
+    only:
+        - master
+        - clustering_deployment
```

Comment:
This should be removed before merging into master

Reply:
removed